integreat_cms/api/v3/locations.py

```diff
@@ -71,9 +71,24 @@ def transform_poi_translation(poi_translation: POITranslation) -> dict[str, Any]
 
     poi = poi_translation.poi
 
-    primary_contact = Contact.objects.filter(
-        location=poi, area_of_responsibility=""
-    ).first()
+    contacts = Contact.objects.filter(location=poi).all()
+    contact_data = []
+    for contact in contacts:
+        contact_data.append(
+            {
+                "id": contact.id,
+                "area_of_responsibility": contact.area_of_responsibility
+                if contact.area_of_responsibility
+                else None,
```

Comment:
Isn't this `None` by default as well, like the other fields?

Reply:
I tried this yesterday and it returned an empty string. I had a chat with the app team and they said None would be better than an empty string

Reply:
Hm, strange. Maybe we should ensure in the `TruncatingCharField` that empty strings get converted to `None`? Not sure though if that is the issue.

In any case, that's beyond the scope of this PR, so I will mark as resolved 👍🏼